.gitignore

```
@@ -16,3 +16,4 @@ npm-debug.log*
 /backstop_data/packageDependentMap.json
 .idea
 lerna-debug.log
+yarn-error.log
```

do we need to add that in the git-ignore. I guess hiding error-logs can't hurt